packages/vaadin-combo-box/test/overlay-position.test.js

```diff
@@ -37,8 +37,9 @@ describe('overlay position', () => {
     comboBox = fixtureSync(`<vaadin-combo-box label='comboBox' style='width: 300px;' items='[1]'></vaadin-combo-box>`);
     const comboBoxRect = comboBox.getBoundingClientRect();
     comboBox.items = makeItems(20);
+    inputField = comboBox.shadowRoot.querySelector('[part="input-field"]');
     dropdown = comboBox.$.dropdown;
-    overlay = dropdown.$.overlay;
+    overlayPart = dropdown.$.overlay.$.overlay;
```

Comment:
Changed `overlay.$.overlay` to `overlayPart` to make the tests a bit more readable.